_layouts/base.html

```diff
@@ -138,7 +138,7 @@ <h1 class="usa-display">{{ page.title }}</h1>
 
     {{ content }}
 
-      <footer class="usa-footer usa-footer--slim" role="contentinfo" aria-label="GSA footer">
+      <footer class="usa-footer usa-footer--slim usa-footer--slim__z-index" role="contentinfo" aria-label="GSA footer">
```

Comment:
Do we need a custom class or would it be enough to use a combination of `position` and `z-index` utility classes?

* https://designsystem.digital.gov/utilities/display/#position
* https://designsystem.digital.gov/utilities/display/#z-index

Reply:
yes, good call!